getting-started/client-blazor.md

```diff
@@ -28,28 +28,28 @@ This article explains how to get the Telerik UI for Blazor components in your **
 @[template](/_contentTemplates/common/get-started.md#blazor-tutorial-intro)
 
 
-For a client-side `Blazor App`, we recommend the usage of **`Blazor (ASP.NET Hosted)`** project. 
+To create a client-side Blazor app, use an **ASP.NET Core hosted** Blazor project:
 @[template](/_contentTemplates/common/get-started.md#project-creation-part-1)
 
-1. Choose the `Blazor WebAssembly App` project type, select the `ASP.NET Core hosted` checkbox, and click `Create`.
+1. Choose the **Blazor WebAssembly App** project type, select the **ASP.NET Core hosted** checkbox, and click **Create**.
 
     ![Select Blazor Project Type](images/choose-project-template.png)
 
-
+@[template](/_contentTemplates/common/get-started.md#project-creation-cli)
 
 ## Step 2 - Add the Telerik Blazor Components to an Existing Project
 
 @[template](/_contentTemplates/common/get-started.md#add-nuget-feed)
 
 @[template](/_contentTemplates/common/get-started.md#get-access)
 
-    1. Right-click on the `Client` project in the solution and select `Manage NuGet Packages`:
```

Comment:
Use bold when referring to UI elements. See https://testdocs.telerik.com/style-guide/element-formatting